BCP195bis/draft-ietf-uta-rfc7525bis.md

```diff
@@ -796,27 +798,17 @@ To address these concerns, TLS implementations SHOULD NOT use static DH keys and
+* {{?RFC7633}} defines a certificate extension that indicates that clients must expect stapled OCSP responses for the certificate and must abort the handshake ("hard-fail") if such a response is not available.
+* OCSP stapling as used in TLS 1.2 does not extend to intermediate certificates within a certificate chain. The Multiple Certificate Status extension {{?RFC6961}} addresses this shortcoming, but it has seen little deployment and had been deprecated by {{RFC8446}}. As a result, we no longer recommend this extension for TLS 1.2.
+* TLS 1.3 ({{Section 4.4.2.1 of RFC8446}}) allows the association of OCSP information with intermediate certificates by using an extension to the CertificateEntry structure. However using this facility remains impractical because many CAs either do not publish OCSP for CA certificates or publish OCSP reports with a lifetime that is too long to be useful.
+* Both CRLs and OCSP depend on relatively reliable connectivity to the Internet, which might not be available to certain kinds of nodes. A common example is newly provisioned devices that need to establish a secure connection in order to boot up for the first time.    
 
-* OCSP stapling as defined in {{!RFC6066}} does not extend to intermediate certificates used in a certificate chain. Although the Multiple Certificate Status extension {{?RFC6961}} addresses this shortcoming, it is a recent addition without much deployment.
-
-* Both CRLs and OCSP depend on relatively reliable connectivity to the Internet, which might not be available to certain kinds of nodes (such as newly provisioned devices that need to establish a secure connection in order to boot up for the first time).
-      
-
-With regard to common public key certificates, servers SHOULD support the following as a best practice given the current state of the art and as a foundation for a possible future solution:
-
-1. OCSP {{?RFC6960}}
-
-1. Both the status\_request extension defined in {{!RFC6066}} and the status_request_v2 extension defined in {{?RFC6961}} (This might enable interoperability with the widest range of clients.)
-
-1. The OCSP stapling extension defined in {{?RFC6961}}
-      
+With regard to common public key certificates, servers SHOULD support the following as a best practice given the current state of the art and as a foundation for a possible future solution: OCSP {{?RFC6960}} and OCSP stapling using the `status_request` extension defined in {{!RFC6066}}. The exact mechanism for embedding the  `status_request` extension differs between TLS 1.2 and 1.3. As a matter of local policy, server operators MAY issue must-staple {{RFC7633}} certificates for the server and/or for client authentication, but we recommend to review the operational conditions before deciding on this feature. 
```

Comment:
A small (unrelated) editorial suggestion:

```suggestion
With regard to common public key certificates, servers SHOULD support the following as a best practice given the current state of the art and as a foundation for a possible future solution: OCSP {{?RFC6960}} and OCSP stapling using the `status_request` extension defined in {{!RFC6066}}. Note that the exact mechanism for embedding the `status_request` extension differs between TLS 1.2 and 1.3. As a matter of local policy, server operators MAY issue must-staple {{RFC7633}} certificates for the server and/or for client authentication, but we recommend to review the operational conditions before deciding on this feature. 
```

And a few questions:
* What is meant by "_common_ public key certificates"?
* Is it correct to say that "server operators _issue_ certs"?  Should it be "request their CA to issue ..." instead?
* I don't understand why we need to talk about "client authentication" here?

Reply:
The "note that" idiom is a matter of style, IMO, and more appropriate for academic texts. But I'm OK with your change.

* At the top of the section we mention "the revocation status of common public key certificates". Agree that this term does not make sense at this point any more.
* Your suggestion is a bit too precise, but OK...
* People mentioned specifically use of OCSP to revoke client certs, probably in closed enterprise ecosystems. I think it's an interesting use case.

Reply:
Thanks for your answers!

Regarding the last point:

> * People mentioned specifically use of OCSP to revoke client certs, probably in closed enterprise ecosystems. I think it's an interesting use case.

is this referring to the 3GPP mention in [John Mattsson's email](https://mailarchive.ietf.org/arch/msg/uta/oFMIoizKXnI7M-z7f_30Vx6rHrg/)?  If so, I don't think it is intended that way.

Reply:
> > * People mentioned specifically use of OCSP to revoke client certs, probably in closed enterprise ecosystems. I think it's an interesting use case.
> 
> is this referring to the 3GPP mention in [John Mattsson's email](https://mailarchive.ietf.org/arch/msg/uta/oFMIoizKXnI7M-z7f_30Vx6rHrg/)? If so, I don't think it is intended that way.

No, I'm referring to [Mohit's response](https://mailarchive.ietf.org/arch/msg/uta/7pnwMeeB1e1uVxbPtxQiS9V2WcY/).

Reply:
cool, scratch my comment then.